robot/resources/lib/python_keywords/epoch.py

```diff
@@ -113,3 +117,11 @@ def tick_epoch(shell: Shell, cluster: Cluster, alive_node: Optional[StorageNode]
         gas=1,
     )
     sleep(parse_time(MAINNET_BLOCK_TIME))
+
+
+@allure.step("Tick Epoch and wait for epochs align")
+def tick_epoch_and_wait(shell: Shell, cluster: Cluster, current_epoch: Optional[int],
```

Comment:
why we have the same logic with epochs ticking in two places? here and in the cluster_test_base.py ?